src/views/MakingARequest/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
```

Comment:
This view namespace and file structure hierarchy feels like it can also be improved upon. No bright ideas at the moment. We should keep an eye on this.